bot/modules/roll/commands/roll.js

```diff
@@ -30,11 +35,29 @@ module.exports = class extends Command
     if(!validFormat.test(text))
       return context.chat('Invalid format.');
 
+    let label = rawText.replace(rollFormatRegex, '');
+    const labelRegexValue = label
+      .toLowerCase()
+      .split('')
+      .map(letter => {
+        const escapables = [
+          '.', '^', '$', '*', '+', '?', '(', ')', '[', '{', '\\', '|'
```

Comment:
Baka pwede 'to 1 line pero kung mas okay ka dito oks lang.

Reply:
Mag exceed to sa 100 lines pag naka 1 line eh.